gdsfactory/functions.py

```diff
@@ -282,7 +283,7 @@ def _inner_decorator(component: ComponentFactory) -> ComponentFactory:
             component: The input function creating a component.
         """
 
-        @cell_with_child
+        @cell_with_child(cache=False)
```

Comment:
this looks dangerous,

do we need it?

Reply:
The tests seemed to fail for me without this so, I guess otherwise the wrapper  is cached